### DISCUSSION TITLE
ERRO NA IMPRESSÃO DE VENDA

### DISCUSSION THREAD

**Discussion by thaleshlopes** [RamonSilva20/mapos#2157] [General]
Pessoal, boa noite! 
Uso e recomendo o MAPOS a um bom tempo. Recentemente perdemos o servidor linux no qual estava instalado uma versão estável dele, pra agilizar a instalação fiz com um XAMPP e instalei a nova versão dele! 

Na hora de salvar a venda em PDF pra mandar pro cliente, ele está adicionando LINHAS antes dos produtos.
Segue uma imagem de como está : [https://ibb.co/mDkry6X](url)

Gostaria de saber se é normal, pois em outras versões não tinha essas linhas.

**Comment by rafaelbrasileiro** [comment]
Boa noite. Sim, essas linhas aparecem agora.

**Reply by thaleshlopes** [reply]
Aparentemente é uma linha para cada item da VENDA! É possível remover?

**Reply by rafaelbrasileiro** [reply]
Sim, é possível.

**Reply by thaleshlopes** [reply]
Pode me orientar como?

**Comment by thaleshlopes** [comment]
Alguém consegue me ajudar a remover essas linhas? Obrigado desde já.

**Reply by thaleshlopes** [reply]
Para remover as linhas bastar ir até a div //div style="margin-top: 0; padding-top: 0"// e remover o //echo '<//hr />'//; do //tbody!
Obrigado a todos!
